toolboxes/pyfeelpp-toolboxes/tests/test_cfpdes.py

```diff
@@ -25,20 +25,20 @@ def test_cfpdes(prefix,case,casefile,dim):
     return not f.checkResults()
     
 
-def test_cfpde_nlthermoelectric():
-    prefix, case, casefile, dim= ( 'thermoelectric', 'ElectroMagnets_HL-31_H1', 'HL-31_H1.cfg', 3)
-    feelpp.Environment.changeRepository(
-        directory="toolboxes/coefficientformpdes/thermoelectric/ElectroMagnets_HL-31_H1")
-    feelpp.Environment.setConfigFile('cfpdes/{}/{}/{}'.format(prefix, case, casefile))
-    linear_case_t = cfpdes(dim=dim)
-    simulate(linear_case_t)
-    assert linear_case_t.checkResults()
-    # now run the nonlinear model using the linear solution as initial guess
-    casefile = 'HL-31_H1_nonlinear.cfg'
-    feelpp.Environment.setConfigFile('cfpdes/{}/{}/{}'.format(prefix, case, casefile))
-    nl_case_t = cfpdes(dim=dim)
-    simulate(nl_case_t)
-    return not nl_case_t.checkResults()
+# def test_cfpde_nlthermoelectric():
```

Comment:
why ?

Reply:
currently, can not work in seq and parallel (sometimes work, sometimes not). I have also a strange behavior when the number of proc increase, maybe a bug somewhere, to investigate

Reply:
it was working in the automated tests via ctest in both seq and par cases.
when did you see the problem ?

Reply:
because the mesh is repartitioned and this partitioning is not always the same. The field loaded can be wrong (something crash if dofs are not compatible). Currently, in sequential we use a .msh and in // a partitioned mesh .json, both case can't be take into account in one config.
Another issue is when the number of proc increases (for example > 20 ), there is something wrong in the automatic diff I think but not very clear, I need to continue my investigation.